### PR TITLE
Add no-op for test and release compile

### LINKED ISSUE #34: Add "no-op" for "testCompile"
Similar to leak canary, can we add a "no-op"?

### PR DESCRIPTION
**Pull request by jaredsburrows**
Fixes #34 
- Created "no-op" module similar to LeakCanary
- You can add the "no-op" to `relaseCompile` and `testCompile` and the original in `debugCompile`
- Updated README.md to show how to use library effectively
- I also removed useless icons in the library, that just adds size to the AAR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ In your `build.gradle`:
 
 ```gradle
  dependencies {
-   debugCompile 'com.github.brianPlummer:tinydancer:0.0.9'
+   debugCompile "com.github.brianPlummer:tinydancer:0.0.9"
+   releaseCompile "com.github.brianPlummer:tinydancer-noop:<latest>"
```
Comment (jaredsburrows) [inline README.md]: @digitalbuddha You will need to publish a new version, maybe a snapshot first?

File: example/build.gradle [hunk 2]
```diff
@@ -11,23 +11,32 @@ android {
...
 }
 
 dependencies {
-    //compile against maven central repo
-    //compile "com.github.brianPlummer:tinydancer:0.0.1"
-
-    //compile against local library
-    compile project(":tinydancer")
+    debugCompile project(":tinydancer")
+    releaseCompile project(":tinydancer-noop")
```
Comment (jaredsburrows) [inline example/build.gradle]: Use the local library to make sure CI stays up to date.

File: tinydancer-noop/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+POM_NAME=com.codemonkeylabs.tinydancer
+POM_ARTIFACT_ID=tinydancer-noop
```
Comment (jaredsburrows) [inline tinydancer-noop/gradle.properties]: Make sure we use "-noop" in the new artifact name.

File: tinydancer-noop/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.codemonkeylabs.fpslibrary.noop">
```
Comment (jaredsburrows) [inline tinydancer-noop/src/main/AndroidManifest.xml]: And in the package name.

File: README.md (hunk 1 above)
Comment (digitalbuddha) [inline README.md]: Yup @brianPlummer will do it this weekend.

File: tinydancer-noop/proguard-rules.pro [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```
Comment (digitalbuddha) [inline tinydancer-noop/proguard-rules.pro]: Does no op need proguard rules?

**Comment by digitalbuddha** [comment]
@jaredsburrows i gotta ask was today a 20% day?

File: tinydancer-noop/proguard-rules.pro (hunk 5 above)
Comment (jaredsburrows) [inline tinydancer-noop/proguard-rules.pro]: It was just the "auto generated" file that came with the new module. There is nothing in it.

**Comment by jaredsburrows** [comment]
@digitalbuddha Haha. Yesterday was but it is Friday and I like to context switch. I want to be able to use this in a few projects for quality checks!

**Comment by digitalbuddha** [comment]
@jaredsburrows thank you for your the contributions.  If you're ever in NY, I'll buy you a beer. Incidentally, my work window faces the microsoft building.  Cheers man and enjoy your weekend.

**Comment by jaredsburrows** [comment]
@digitalbuddha Thanks! I appreciate your hard work on this. I'll continue to add updates!

**Pull request merged**